docs/recipes/DevelopingOnFork.md

```diff
@@ -0,0 +1,201 @@
+---
+sidebar_position: 6
+title: Working with a Forked Network
+description: Learn how to develop and test your dApp using a fork of mainnet network with custom wallets.
+---
+
+# Working with a Forked Network
+This recipe shows how to run a local fork of mainnet and interact with it using Scaffold Stark's contract hooks.
+
+<details open>
+<summary>Here is the complete code for interacting with a forked network:</summary>
+
+```tsx title="components/ForkNetworkInteraction.tsx"
+import { useAccount } from "@starknet-react/core";
+import { useScaffoldContract, useScaffoldWriteContract } from "~~/hooks/scaffold-stark";
+import { notification } from "~~/utils/scaffold-stark";
+
+export const ForkNetworkInteraction = () => {
+  const { address } = useAccount();
+  
+  // Get contract instance
+  const { data: contract, isLoading: isContractLoading } = useScaffoldContract({
+    contractName: "YourContract",
+  });
+
+  // Set up write function
+  const { sendAsync: setGreeting, isPending } = useScaffoldWriteContract({
+    contractName: "YourContract",
+    functionName: "set_greeting",
+    args: ["Testing Fork!"],
+  });
+
+  const handleSetGreeting = async () => {
+    try {
+      await setGreeting();
+      notification.success("Greeting updated!");
+    } catch (error) {
+      console.error("Error:", error);
+      notification.error("Failed to update greeting");
+    }
+  };
+
+  return (
+    <div className="bg-base-300 p-6 rounded-lg max-w-md mx-auto mt-6">
+      <h2 className="text-lg font-bold mb-4">Fork Network Interaction</h2>
+      <div className="space-y-4">
+        <div className="text-sm">
+          <p>Network: Starknet Devnet (Fork)</p>
+          <p>Your Address: {address}</p>
+          <p>Contract: {contract?.address}</p>
+        </div>
+        <button 
+          className="btn btn-primary"
+          onClick={handleSetGreeting}
+          disabled={isPending || isContractLoading}
+        >
+          {isPending ? "Sending..." : "Test Greeting"}
+        </button>
+      </div>
+    </div>
+  );
+};
+```
+</details>
+
+## Implementation Guide
+
+### Step 1: Start the Fork Network
+First, start a local fork of mainnet:
+
+```bash title="terminal"
+# Start fork
+yarn chain --fork-network https://starknet-mainnet.public.blastapi.io/rpc/v0_7
+
+# In a new terminal, deploy contracts
+yarn deploy
+
+# Start frontend on another terminal
+yarn start
+```
+
+### Step 2: Create Network Interaction Component
```

Comment:
we dont need this section the important part is forking, deploying, and connecting with the burner / extension to interact on `/debug-ui` without adding extra components.